### ISSUE TITLE
High training loss when using M-ROPE

### ISSUE THREAD

**Issue by Wiselnn570** [QwenLM/Qwen2-VL#406]
I tried incorporating the M-ROPE module from Qwen2-VL into my own model to train on video data, but I noticed that the loss is quite high during training. With a total batch size of 128, the loss hovers between 2 and 3. Is this a normal phenomenon?

**Comment by tang-bd** [comment]
Hi, did you solve this problem?

**Comment by Wiselnn570** [comment]
Yep

Bingda Tang ***@***.***> 于2024年10月30日周三 23:02写道：

**Comment by tang-bd** [comment]
How did you solve the problem? I also have some issues using M-RoPE.

**Comment by Wiselnn570** [comment]
> How did you solve the problem? I also have some issues using M-RoPE.

When using LoRA before, the loss was very high, but it normalized after fully unfreezing the language model.

**Comment by tang-bd** [comment]
Thanks!
